### ISSUE TITLE
Questions: ev3dev2 and gears

### ISSUE THREAD

**Issue by DJuego** [QuirkyCort/gears#99]
Hey! After a long time I have been missing, here I am again. :-)

We want to dedicate this edition of our educational project to the simulator/real world comparison. For the first time, we will work with **Gears** and EV3 units in parallel. This is something we have already done with Open Roberta.

At the moment I have run for the first time the code base generated by **Gears** on an EV3 brick. In fact, this code:

```python

#!/usr/bin/env python3

# Import the necessary libraries
import time
import math
from ev3dev2.motor import *
from ev3dev2.sound import Sound
from ev3dev2.sensor import *
from ev3dev2.sensor.lego import *
from ev3dev2.sensor.virtual import *

# Create the sensors and motors objects
motorA = LargeMotor(OUTPUT_A)
motorB = LargeMotor(OUTPUT_B)
left_motor = motorA
right_motor = motorB
tank_drive = MoveTank(OUTPUT_A, OUTPUT_B)
steering_drive = MoveSteering(OUTPUT_A, OUTPUT_B)

spkr = Sound()
radio = Radio()

color_sensor_in1 = ColorSensor(INPUT_1)
ultrasonic_sensor_in2 = UltrasonicSensor(INPUT_2)
gyro_sensor_in3 = GyroSensor(INPUT_3)
touch_sensor_in4 = TouchSensor(INPUT_4)
# Here is where your code starts

```

Apparently it works! but i need comment the lines:

_from ev3dev2.sensor.virtual import *_

and

_radio = Radio()_

My question is: 

What is the best way to work with gears and the ev3dev brick?  Is there a modified version of the ev3dev2 API (see [link](https://github.com/QuirkyCort/gears/tree/master/public/ev3dev2)) that should be installed? Or is there an option to generate only compatible code?

Thank you very much for **Gears**!

DJuego

PS: About this;  Is there a clever way to upload Gears files to ev3dev with minimal user assistance? :-) (My environment is Windows 10 x64)
